example-spec.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Specification Name
+layout: spec
+work-in-progress: true
+copyrights:
+  -
+    name: "Your Name"
+    email: "you@yourdomain.com"
+    period: "Year Range (e.g. 2010-2016)"
+---
+
+This is an example specification, which is intended to help you while writing a new proposal!
+
+Feel free to use this as a rough guideline while writing up your specification. However,
+keep in mind that the sections here are not set in stone.
+
+Also take a look at other specifications and the CONTRIBUTING.md file for further
+suggestions and information to keep in mind while writing a proposal.
+
+Some notes:
+
+- New capability and tag names should be listed in the _"Notes for implementing work-in-progress version"_ section, with the appropriate disclaimer and `draft/` prefix, until it's ratified.
+
+---
+
+## Notes for implementing work-in-progress version
+
+This is a work-in-progress specification.
+
+Software implementing this work-in-progress specification MUST NOT use the
+unprefixed `example-spec` capability name. Instead, implementations SHOULD
+use the `draft/example-spec` capability name to be interoperable with other
+software implementing a compatible work-in-progress version.
+
+The final version of the specification will use an unprefixed capability name.
+
+
+## Introduction
+
+This feature is intended to solve problem A, or provide a new feature B.
+
+We've written this specification because of C.
+
+In other words, this section contains a rough overview of why the spec exists, the new features you're trying to
+provide and/or the issues you're trying to solve. In other words, why other people should be
+motivated to implement it, and why you have been motivated to write it.
+
+## Architecture
+
+This specification introduces various capabilities, messages, numerics and tokens as
+necessary. This section (and subsections) describe exactly how those work, and define those
+new or extended caps/messages/numerics/tokens.
+
+The following sections may be useful to present, as necessary:
+
+### Capabilities
+
+New capabilities that this spec defines are listed here.
+
+### Tags
+
+New tags that this spec defines are listed here.
+
+### Messages
+
+New IRC messages (i.e. `PRIVMSG`/`PING`/`CHGHOST`) that this spec defines are listed here.
+
+### Numerics
+
+New numerics that this spec defines are listed here. For example:
+
+| No. | Label          | Format                                                                  |
+| --- | ---------------| ----------------------------------------------------------------------- |
+| 001 | `RPL_WELCOME`  | `<client> :Welcome to the Internet Relay Network <nick>!<user>@<host> ` |
+| 002 | `RPL_YOURHOST` | `<client> :Your host is <servername>, running version <version> `       |
+
+### RPL_ISUPPORT Tokens
+
+`RPL_ISUPPORT` tokens that this spec defines are listed here.
```

Comment:
If there isn't any (like most specs) this section should be omitted. This is similar to the alternatives section which says omit if empty. Thoughts?